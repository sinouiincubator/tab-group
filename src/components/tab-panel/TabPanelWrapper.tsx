import styled from 'styled-components';

const TabPanelWrapper = styled.div`
  width: 100%;
  flex-shrink: 0;
`;

export default TabPanelWrapper;
