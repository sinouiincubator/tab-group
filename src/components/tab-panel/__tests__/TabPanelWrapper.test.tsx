import React from 'react';
import renderer from 'react-test-renderer';
import TabPanelWrapper from '../TabPanelWrapper';

it('渲染 TabPanelWrapper', () => {
  const tree = renderer.create(<TabPanelWrapper />).toJSON();

  expect(tree).toMatchSnapshot();
});
