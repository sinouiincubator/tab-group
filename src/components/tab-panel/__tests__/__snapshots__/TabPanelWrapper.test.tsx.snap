// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`渲染 TabPanelWrapper 1`] = `
<div
  className="sc-bdVaJa UGpYE"
/>
`;
